signing_service/src/signing_service.py

```diff
@@ -0,0 +1,16 @@
+import logging.config
+import os
+
+
+logger = logging.getLogger()
+crash_logger = logging.getLogger('crash')
+
+
+def main():
+    while True:
+        pass
+
+
+if __name__ == '__main__':
+    logging.config.fileConfig(os.path.join('..', 'logging.ini'))
```

Comment:
This will load `logging.ini` from a directory relative to the working directory (the one where you run the script from). I think it should be relative to the place where the script is located instead.

Reply:
:+1: done